src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -27,17 +26,51 @@ const ActorInit Bg_Inibs_Movebg_InitVars = {
     (ActorFunc)BgInibsMovebg_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96578[] = {
+s32 D_80B96560[] = { 0x060062D8, 0x06001DC0 };
+s32 D_80B96568[] = { 0x06006140, 0x06001C10 };
+s32 D_80B96570[] = { 0x06006858, 0x06002598 };
```

Comment:
```suggestion
Gfx* D_80B96560[] = { 0x060062D8, 0x06001DC0 };
Gfx* D_80B96568[] = { 0x06006140, 0x06001C10 };
AnimatedMat* D_80B96570[] = { 0x06006858, 0x06002598 };
```
Again the symbols can be deduced from the function calls. Should also convert the raw hex addresses into symbols. They would also needed to be added to undef_syms if they are not already there. For example of this you can take a look at `ObjEnding` which is actually really similar to this.